damus/ContentView.swift

```diff
@@ -168,7 +168,20 @@ struct ContentView: View {
                 EmptyView()
             }
         }
-        .navigationBarTitle(selected_timeline == .home ?  "Home" : "Global", displayMode: .inline)
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .principal) {
+                if selected_timeline == .home {
+                    Image("damus-home")
+                    .resizable()
+                    .frame(width:30,height:30)
+                    .shadow(color: Color("DamusPurple"), radius: 2)
+                } else {
+                    Text("Global")
```

Comment:
@BenGWeeks @jb55 

I think this line introduced a bug where it shows `Global` instead of `DM` in the toolbar for the DM view.

I'm working on a PR to fix this now.

Reply:
https://github.com/damus-io/damus/pull/321